packages/vpt/vpt.5.0.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "unixjunkie@sdf.org"
+authors: ["Francois BERENGER"]
+homepage: "https://github.com/UnixJunkie/vp-tree"
+bug-reports: "https://github.com/UnixJunkie/vp-tree/issues"
+dev-repo: "git+https://github.com/UnixJunkie/vp-tree.git"
+license: "BSD-3-Clause"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "2.8"}
```
The `build` tag is not supported by dune. See https://github.com/ocaml/dune/issues/2147
Also your `dune-project` file states that this package required at least dune 2.8